### DISCUSSION TITLE
How to Upgrade?

### DISCUSSION THREAD

**Discussion by romanr** [nagadomi/nunif#188] [Q&A] [answered]
I downloaded the `nunif-windows.zip` again but it's identical to pervious one. I have installed iw3 month ago, and read saw there there were improvements made.   
Or should I check out git repo?

**Comment by nagadomi** [comment, marked as answer]
When you run `update.bat`,  `nunif` folder will be updated to the latest version on Github.

**Reply by romanr** [reply]
thank you!
